composer.json

```diff
@@ -54,7 +54,7 @@
         "drupal/memcache": "^2.5.0",
         "drupal/metatag": "^1.26",
         "drupal/moderation_sidebar": "^1.4",
-        "drupal/paragraphs": "^1.15",
+        "drupal/paragraphs": "^1.16",
```

Comment:
Add "(#4134) Multiple warning messages when having untranslatable fields" to the claro update commit.